.gitignore

```diff
@@ -0,0 +1 @@
+.vscode/*
```

Comment:
Is this file needed?

Reply:
`.gitignore` is good to have so we don't pollute the repository with non-relevant files.

Reply:
IMO the  committed `.gitignore` file should include names of files/directories that are created when working on the code directly and should not be committed.

It should not include names of files that are only created by personal tools such as vscode, GoLand, intelliJ etc. For these, we can use a system-wide [Global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)